### ISSUE TITLE
Regression: Reviewer can see other scores

### ISSUE THREAD

**Issue by lod** [zookeepr/zookeepr#419]
Identified by zkpylons/tests/functional/test_review.py :test_reviews_isolated

To reproduce:
Create two reviewer accounts and one proposal.
Review proposal with reviewer 1.
Log in with reviewer 2.

Expected:
Reviewer 2 can't see the score from reviewer 1.

**Comment by noisymime** [comment]
I had thought it was intended behaviour that one reviewer could see another's scoring/comments?

**Comment by lod** [comment]
Possibly, having a GUI section to list them doesn't make much sense if it isn't.

However there was a specific test to ensure that it didn't happen, so I felt reluctant to delete or modify the test without any sort of documentation on there being a change.

**Comment by jhesketh** [comment]
This is known functionality and I haven't heard the papers committee complain/ask for scores to be hidden.

A separate issue is that we should probably raise a warning when the paper you are reviewing is proposed by somebody with the reviewer role and therefore can see the votes+comments.

**Comment by mikalstill** [comment]
Yeah, I am surprised we're talking about hiding scores. Certainly no one has talked to me about it and I am not sure its a good idea. Perhaps if it was optional?

**Comment by lod** [comment]
Thanks for the feedback, I'll update the test to enforce the current visible behaviour

**Comment by stevewalsh** [comment]
I wonder if this is related to the GreaseMonkey scripts that Puzzlement wrote for reviewers a while ago, and were still in use as of 2014

"Last year, I wrote some support scripts for reviewing proposals. Specifically,
we're usually asked to ignore travel requests when scoring reviews, and many
people WANT to do at least a first pass while ignoring previous reviews!

But ZooKeepr shows you this information, right there on the review page!

I wrote three Greasemonkey scripts that hides them behind a "click to see" kind
of thing..."
